packages/forms/src/Brazil/FormFieldPhone.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  FormFieldPatternFormat,
+  FormFieldPatternFormatProps,
+} from '../FormFieldPatternFormat';
+
+export type FormFieldPhoneProps = {
+  label: string;
+  name: string;
+} & Partial<FormFieldPatternFormatProps>;
+
+export const FormFieldPhone = ({
+  label,
+  name,
+  ...formFieldPatternFormatProps
+}: FormFieldPhoneProps) => {
+  return (
+    <FormFieldPatternFormat
+      name={name}
+      label={label}
+      format={'(##) #####-####' || '(##) ####-#####'}
```

Comment:
Phone format is not working for 8 digits.

![image](https://github.com/ttoss/ttoss/assets/16626980/03ad1063-37ef-4205-9a2d-1f1146e7144f)